.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,152 @@
+    - get-pr-number
+    - bundle-container-push-pr
+    if: always() # remove this after we fix signing
+
+    permissions:
+      packages: write
+      contents: read
+
+    outputs:
+      image-digest: ${{ steps.container_info.outputs.image-digest }}
+      image-tags: ${{ steps.container_info.outputs.image-tags }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.6.0
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ghcr.io/complianceascode/compliance-operator-catalog
+          tags: |
+            type=raw,value=${{ needs.get-pr-number.outputs.pr-number }}
+            type=sha,format=long
+          labels: |
+            org.opencontainers.image.source=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}
+            org.opencontainers.image.title=compliance-operator-catalog
+            org.opencontainers.image.version=latest
+            org.opencontainers.image.licenses='Apache-2.0'
+            org.opencontainers.image.vendor='Compliance Operator Authors'
+        
+      - name: Generate catalog dockerfile
+        id: generate_catalog_dockerfile
+        run: |
+          make catalog-docker BUNDLE_IMGS=ghcr.io/complianceascode/compliance-operator-bundle:${{ needs.get-pr-number.outputs.pr-number }}
+
+      - name: Build container images and push
+        id: docker_build
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: catalog.Dockerfile
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          push: true
+          platforms: 'linux/amd64'
+
+      - name: Get container info
+        id: container_info
+        run: |
+          image_tags="${{ needs.get-pr-number.outputs.pr-number }}"
+          echo "::set-output name=image-digest::${{ steps.docker_build.outputs.digest }}"
+          echo "::set-output name=image-tags::${image_tags}"
+
+  comment-pr:
+    needs: 
+    - operator-container-push-pr
+    - bundle-container-push-pr
+    - openscap-container-push-pr
+    - catalog-container-push-pr
+    - get-pr-number
+    runs-on: ubuntu-latest
+    name: Comment on the PR
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            :robot: To deploy this PR, run the following command:
+            ```
+            make catalog-deploy BUNDLE_IMGS=ghcr.io/complianceascode/compliance-operator-bundle:${{ needs.get-pr-number.outputs.pr-number }}
```

Comment:
It looks like `BUNDLE_IMGS` is only used within `catalog-image` and `catalog-docker`?